main.go

```diff
@@ -6,22 +6,24 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net"
 	"os"
+	"os/signal"
+	"syscall"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
+	pb "sigs.k8s.io/secrets-store-csi-driver/provider/v1alpha1"
 )
 
 var (
-	attributes = pflag.String("attributes", "", "volume attributes")
-	secrets    = pflag.String("secrets", "", "node publish ref secret")
-	targetPath = pflag.String("targetPath", "", "Target path to write data.")
-	permission = pflag.String("permission", "", "File permission")
-	debug      = pflag.Bool("debug", false, "sets log to debug level")
-	version    = pflag.Bool("version", false, "prints the version information")
+	endpoint = pflag.String("endpoint", "/tmp/vault.sock", "path to socket on which to listen for driver gRPC calls")
```

Comment:
Should this default to `/provider/vault.sock`?

Reply:
It depends how you run it - I chose `/tmp/vault.sock` to make it easier to run locally as `/tmp` will (always?) already exist, but when crafting yaml to deploy the provider, users should match up the volume mount they create and this argument. Obviously the helm chart will get those two married up correctly by default.